pkg/backend/pull.go

```diff
@@ -49,7 +49,21 @@ func (b *backend) Pull(ctx context.Context, target string, cfg *config.Pull) err
 
 	manifestDesc, manifestReader, err := src.Manifests().FetchReference(ctx, tag)
 	if err != nil {
-		return fmt.Errorf("failed to fetch the manifest: %w", err)
+		// fallback to fetch the manifest without proxy.
```

Comment:
You cannot get manifest via a proxy because they will be cached the content of the manifest. If the content of the manifest changed, you will get the wrong content.